### PR TITLE
convert to @jbolda/gatsby-themes

### PR DESCRIPTION
**Pull request by jbolda**

### CODE REVIEW

**Comment by jbolda** [comment]
We seem to be plagued by a regression as noted in https://github.com/gatsbyjs/gatsby/issues/20150. We will just use a webpack config temporarily until a fix is implemented.

File: src/articles/0006-choropleth-on-d3v4/choropleth.js [hunk 1]
```diff
@@ -62,39 +36,12 @@ class choroplethBase extends React.Component {
...
-          },
-          {
-            wrapper: "column",
-            uniqueKey: "text-column",
-            renderComponent: this.renderText.bind(this)
-          }
-        ]}
-        {...this.props}
-      />
+      <React.Fragment>
```
Comment (RyanWarner) [inline src/articles/0006-choropleth-on-d3v4/choropleth.js]: You could simplify this a bit with
```
<>
  <div id="states" />
  <div id="tooltip" />
</>
```

Comment (jbolda) [reply src/articles/0006-choropleth-on-d3v4/choropleth.js]: I prefer to be explicit with that as it's really hard to search on find what `<>` does if you don't already know.

**Comment by jbolda** [comment]
There are a few things that could use some love, but this is an improvement enough that it is worth merging in.

**Pull request merged**